packages/wdio-config/src/lib/ConfigParser.js

```diff
@@ -174,9 +175,18 @@ export default class ConfigParser {
 
             // Allow --suite and --spec to both be defined on the command line
             // Removing any duplicate tests that could be included
-            const tmpSpecs = spec.length > 0 ? [...specs, ...suiteSpecs] : suiteSpecs
+            let tmpSpecs = spec.length > 0 ? [...specs, ...suiteSpecs] : suiteSpecs
+
+            if (Array.isArray(capSpecs)) {
+                tmpSpecs = tmpSpecs.concat(ConfigParser.getFilePaths(capSpecs))
+            }
+
+            if (Array.isArray(capExclude)) {
+                exclude = exclude.concat(ConfigParser.getFilePaths(capExclude))
+            }
 
-            return [...new Set(tmpSpecs)]
+            specs = [...new Set(tmpSpecs)]
+            return specs.filter(spec => exclude.indexOf(spec) < 0)
```

Comment:
Can be replaced with !exclude.inlcudes(spec)

Reply:
Sure. The same syntax is used outside of the suite loop too. I will update both